keyboards/kippersun/nanetta65/config.h

```diff
@@ -0,0 +1,41 @@
+/* Copyright 2021 Kipper Sun
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+
+/* key matrix size */
+#define MATRIX_ROWS 5
+#define MATRIX_COLS 15
+
+/* key matrix pins */
+#define MATRIX_ROW_PINS { F4, F1, F0, F6, B0 }
+#define MATRIX_COL_PINS { B1, B3, D0, D1, D2, D3, D5, D4, D6, D7, B4, B5, F7, F5, B6 }
+
+/* COL2ROW or ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCE 5
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+#define RGB_DI_PIN C6
+#define RGB_MATRIX_LED_COUNT 1
```

Comment:
This all should be migrated to `info.json`, and ideally [`RGBLIGHT`](https://docs.qmk.fm/#/feature_rgblight) feature should be used, since it's only one LED.
ps. You can use `qmk info -f json -kb kippersun/nanetta65` to help.